src/buildercore/terraform.py

```diff
@@ -11,7 +11,7 @@
 only_if_managed_services_are_present = only_if(*MANAGED_SERVICES)
 
 EMPTY_TEMPLATE = '{}'
-PROVIDER_FASTLY_VERSION = '0.4.0',
+PROVIDER_FASTLY_VERSION = '0.9.0',
```

Comment:
New `activate` option means there's _always_ a change to apply... 🤔

Reply:
Hmm, doesn't seem to always be true.

Reply:
Think this just applies when a service hasn't been updated with this new version. Can't see it happening again after that.